addons/website_event_track/static/src/js/event_track_search.js

```diff
@@ -0,0 +1,16 @@
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.websiteEventSearchTrack = publicWidget.Widget.extend({
+    selector: '.o_wevent_event_track_search_box',
+    events: {
+        'search .search-query': '_onSearch',
+    },
+
+    _onSearch: function () {
+        const input = this.el.querySelector('input.search-query');
+        if(!input.value) {
+            const form = this.el.querySelector('form.o_wevent_event_searchbar_form');
+            form.submit();
```

Comment:
Defining a form constant is unnecessary here as we're not doing any check on it.
Maybe something like this will work? :slightly_smiling_face: 
`this.el.querySelector('form.o_wevent_event_searchbar_form')?.submit()`

Reply:
Hello @amdi-odoo 

Changes are done.
Can you please check it again?

Thanks :slightly_smiling_face: